TTS/tts/utils/text/korean/ko_dictionary.py

```diff
@@ -0,0 +1,171 @@
+# coding: utf-8
+
+etc_dictionary = {
+    "2 30대": "이삼십대",
+    "20~30대": "이삼십대",
+    "20, 30대": "이십대 삼십대",
+    "1+1": "원플러스원",
+    "3에서 6개월인": "3개월에서 육개월인",
+}
+
+english_dictionary = {
+    "Devsisters": "데브시스터즈",
+    "track": "트랙",
+    # krbook
+    "LA": "엘에이",
+    "LG": "엘지",
+    "KOREA": "코리아",
+    "JSA": "제이에스에이",
+    "PGA": "피지에이",
+    "GA": "지에이",
+    "idol": "아이돌",
+    "KTX": "케이티엑스",
+    "AC": "에이씨",
+    "DVD": "디비디",
+    "US": "유에스",
+    "CNN": "씨엔엔",
+    "LPGA": "엘피지에이",
+    "P": "피",
+    "L": "엘",
+    "T": "티",
+    "B": "비",
+    "C": "씨",
+    "BIFF": "비아이에프에프",
+    "GV": "지비",
+    # JTBC
+    "IT": "아이티",
+    "IQ": "아이큐",
+    "JTBC": "제이티비씨",
+    "trickle down effect": "트리클 다운 이펙트",
+    "trickle up effect": "트리클 업 이펙트",
+    "down": "다운",
+    "up": "업",
+    "FCK": "에프씨케이",
+    "AP": "에이피",
+    "WHERETHEWILDTHINGSARE": "",
+    "Rashomon Effect": "",
+    "O": "오",
+    "OO": "오오",
+    "B": "비",
+    "GDP": "지디피",
+    "CIPA": "씨아이피에이",
+    "YS": "와이에스",
+    "Y": "와이",
+    "S": "에스",
+    "JTBC": "제이티비씨",
+    "PC": "피씨",
+    "bill": "빌",
+    "Halmuny": "하모니",  #####
+    "X": "엑스",
+    "SNS": "에스엔에스",
+    "ability": "어빌리티",
+    "shy": "",
+    "CCTV": "씨씨티비",
+    "IT": "아이티",
+    "the tenth man": "더 텐쓰 맨",  ####
+    "L": "엘",
+    "PC": "피씨",
+    "YSDJJPMB": "",  ########
+    "Content Attitude Timing": "컨텐트 애티튜드 타이밍",
+    "CAT": "캣",
+    "IS": "아이에스",
+    "SNS": "에스엔에스",
+    "K": "케이",
+    "Y": "와이",
+    "KDI": "케이디아이",
+    "DOC": "디오씨",
+    "CIA": "씨아이에이",
+    "PBS": "피비에스",
+    "D": "디",
+    "PPropertyPositionPowerPrisonP" "S": "에스",
+    "francisco": "프란시스코",
+    "I": "아이",
+    "III": "아이아이",  ######
+    "No joke": "노 조크",
+    "BBK": "비비케이",
+    "LA": "엘에이",
+    "Don": "",
+    "t worry be happy": " 워리 비 해피",
+    "NO": "엔오",  #####
+    "it was our sky": "잇 워즈 아워 스카이",
+    "it is our sky": "잇 이즈 아워 스카이",  ####
+    "NEIS": "엔이아이에스",  #####
+    "IMF": "아이엠에프",
+    "apology": "어폴로지",
+    "humble": "험블",
+    "M": "엠",
+    "Nowhere Man": "노웨어 맨",
+    "The Tenth Man": "더 텐쓰 맨",
+    "PBS": "피비에스",
+    "BBC": "비비씨",
+    "MRJ": "엠알제이",
+    "CCTV": "씨씨티비",
+    "Pick me up": "픽 미 업",
+    "DNA": "디엔에이",
+    "UN": "유엔",
+    "STOP": "스탑",  #####
+    "PRESS": "프레스",  #####
+    "not to be": "낫 투비",
+    "Denial": "디나이얼",
+    "G": "지",
+    "IMF": "아이엠에프",
+    "GDP": "지디피",
+    "JTBC": "제이티비씨",

```

Comment:
repeat
delete

Reply:
Thank you. :)
I will reflect the corrections you provided.

Reply:
While reflecting the correction you provided, I thought that the contents of the `english_dictionar`y of this `ko_dictionary.py` were of low quality.

So, I'm going to put in only popular words, and later I'm going to add content to dictionary, or I'm going to add a english-korean transliteration function instead of dictionary.